doc/source/theory/fluid_dynamics/vof.rst

```diff
@@ -0,0 +1,233 @@
+The Volume of Fluid (VOF) method
+#############################################
+
+Numerous exemples of flow encountered in engineering involve multiple fluids: sloshing of fuel in aircraft tanks, mixing of bread dough, and motion of droplets and bubbles to name a few. In these cases, the involved fluids can be immiscible, and we are interested in the evolution of the interfaces between the fluids.
```

Comment:
Our VOF implementation really is more like a conservative level set implementation. Maybe it would be time to rename it to CLS instead of VOF?